spec/system/jobseekers_can_add_qualifications_to_their_job_application_spec.rb

```diff
@@ -11,66 +11,78 @@
+        click_on I18n.t("buttons.save_qualification.one")
+        expect(current_path).to eq(jobseekers_job_application_build_path(job_application, :qualifications))
+        expect(page).to have_content("Superteacher Certificate")
+        expect(page).to have_content("Teachers Academy")
+        expect(page).to have_content("I expect to finish next year")
+        expect(page).not_to have_content("Grade")
+        expect(page).not_to have_content("Year")
+      end
     end
 
-    it "allows jobseekers to add a custom qualification or course (category 'other')" do
-      select_qualification_category("Other qualification or course")
-      expect(page).to have_content(I18n.t("jobseekers.job_applications.qualifications.new.heading.other"))
-      validates_step_complete(button: I18n.t("buttons.save_qualification.one"))
-      fill_in_other_qualification
-      click_on I18n.t("buttons.save_qualification.one")
-      expect(current_path).to eq(jobseekers_job_application_build_path(job_application, :qualifications))
-      expect(page).to have_content("Superteacher Certificate")
-      expect(page).to have_content("Teachers Academy")
-      expect(page).to have_content("I expect to finish next year")
-      expect(page).not_to have_content("Grade")
-      expect(page).not_to have_content("Year")
-    end
-
-    it "allows jobseekers to add a common secondary qualification" do
-      select_qualification_category("GCSE")
-      expect(page).to have_content(I18n.t("jobseekers.job_applications.qualifications.new.heading.gcse"))
-      validates_step_complete(button: I18n.t("buttons.save_qualification.other"))
-      fill_in_gcse
-      # TODO: fill_in_another_gcse
-      click_on I18n.t("buttons.save_qualification.other")
-      expect(current_path).to eq(jobseekers_job_application_build_path(job_application, :qualifications))
-      expect(page).to have_content("GCSEs")
-      expect(page).to have_content("Churchill School for Gifted Macaques")
-      expect(page).to have_content("Maths – 110%")
-      expect(page).to have_content("2020")
-    end
-
-    it "allows jobseekers to add a custom secondary qualification" do
-      select_qualification_category("Other secondary qualification")
-      expect(page).to have_content(I18n.t("jobseekers.job_applications.qualifications.new.heading.other_secondary"))
-      validates_step_complete(button: I18n.t("buttons.save_qualification.other"))
-      fill_in_secondary_qualification
-      # TODO: fill_in_another_secondary_qualification
-      click_on I18n.t("buttons.save_qualification.other")
-      expect(current_path).to eq(jobseekers_job_application_build_path(job_application, :qualifications))
-      expect(page).to have_content("Welsh Baccalaureates")
-      expect(page).to have_content("Happy Rainbows School for High Achievers")
-      expect(page).to have_content("Science – 5")
-      expect(page).to have_content("2020")
+    context "when JavaScript is required", js: true do
```

Comment:
This is because of https://ukgovernmentdfe.slack.com/archives/G0127R0KXQF/p1619434351013200

Reply:
Setting a spec to `js: true` seems make the radio button invisible.

Reply:
yeah would be good to get the bottom of this as per slack thread earlier

Reply:
but that doesnt have to be in the scope ofthis PR